selfdrive/modeld/runners/thneedmodel.h

```diff
@@ -0,0 +1,33 @@
+#ifndef THNEEDMODEL_H
+#define THNEEDMODEL_H
```

Comment:
we've been replacing this style header guard with `#pragma once` across the codebase